lib/branches/get-tags.js

```diff
@@ -1,8 +1,9 @@
-import { escapeRegExp, template } from "lodash-es";
+import { escapeRegExp } from "lodash-es";
 import semver from "semver";
 import pReduce from "p-reduce";
 import debugTags from "debug";
 import { getNote, getTags } from "../../lib/git.js";
+import { makeTag } from "../../lib/utils.js";
```

Comment:
I copied the line above, but this could also be:
```suggestion
import { getNote, getTags } from "../git.js";
import { makeTag } from "../utils.js";
```
though in that manner it would lose the mention of libs and might not be as explicit / obvious.